.github/workflows/release-latest.yml

```diff
@@ -3,7 +3,7 @@ name: Release (latest)
 on:
   push:
     branches:
-      - v4.x
+      - v5.x
```

Comment:
You will also need to add a release file for v4 since it will no longer be the latest. See: https://github.com/DataDog/dd-trace-js/commit/fb3b6f6ffd7563b8fe286a483a635b08dcecc50b#diff-fa70e95b81dc4d89f17178f2c4be5c9b54a855a4b2361afd6ae42ad652b8b675